README.md

```diff
@@ -21,16 +25,18 @@ The list of RFCs currently under discussion can be found at: [Active RFC List][]
 
 Each week the [DevTools core team][] will attempt to review some set of open RFC issues. Reviewed RFCs will also be mentioned in the weekly DevTools team meeting (see archives for [devtools-weekly-meetings][]).
```

Comment:
Are these supposed to be actual links?

Reply:
Yes, the link locations are defined at the bottom. I don't usually do that in md files, but that's how links were used in this one so I kept following this pattern.

Reply:
https://github.com/juliandescottes/rfcs/tree/readme `[DevTools core team][]` isn't clickable here.

Reply:
ah! there was a typo in this one, should have been `DevTools RFC team`, good catch.